### PR TITLE
example to use v2 volume mesher

### PR DESCRIPTION
**Pull request by feilin-flexcompute**

### CODE REVIEW

File: examples/beta_stl_volume_mesher_from_inputs.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import flow360 as fl
+from flow360.component.meshing.params import VolumeMeshingParams, Volume,\
+    Farfield
+
+import os
+os.environ["FLOW360_BETA_FEATURES"] = '1'
+
+surface_mesh = fl.SurfaceMesh.create(
```
Comment (feilin-flexcompute) [inline examples/beta_stl_volume_mesher_from_inputs.py:8]: fl.SurfaceMesh.from_file(filename="xxx.stl", params=None, name="xxxx", tags=[], solver_version="")

File: examples/beta_stl_volume_mesher_from_inputs.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    type = "beta",
+    volume = Volume(
+        first_layer_thicknes=0.001,
+        growth_rate=1.1,
+        num_boundary_layers=2,
+    ),
+    farfield = Farfield(type="auto")
+)
+
+volume_mesh = surface_mesh.create_volume_mesh("airplane-volume-mesh-beta", params=params)
```
Comment (feilin-flexcompute) [inline examples/beta_stl_volume_mesher_from_inputs.py:25]: volume_mesh = fl.VolumeMesh.create(surface_mesh = surface_mesh, name="", ...)

File: examples/beta_stl_volume_mesher_from_inputs.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+surface_mesh = fl.SurfaceMesh.create(
+    geometry_file = "airplaneGeometry.stl", params=None, name="airplane-surface-mesh-beta"
+)
+surface_mesh = surface_mesh.submit()
+print(surface_mesh)
+print(surface_mesh.params)
+
+params = fl.VolumeMeshingParams(
+    type = "beta",
```
Comment (feilin-flexcompute) [inline examples/beta_stl_volume_mesher_from_inputs.py:16]: version="v1"

File: flow360/component/surface_mesh.py [hunk 4]
```diff
@@ -57,7 +57,78 @@ def to_surface_mesh(self) -> SurfaceMesh:
         return SurfaceMesh(self.id)
 
 
-class SurfaceMeshDraft(ResourceDraft):
+class SurfaceMeshDraftFromSurfaceMesh(ResourceDraft):
```
Comment (maciej-flexcompute) [inline flow360/component/surface_mesh.py:60]: can you follow VolumeMeshDraft design so we don't have 2 separate classes?

The interface should be:

create surface mesh
```python
surface_mesh = fl.SurfaceMesh.create(
    Airplane.geometry, params=params, name="airplane-new-python-client"
)
```


upload surface mesh
```python
surface_mesh = fl.SurfaceMesh.from_file(
    "file.stl", name="airplane-new-python-client"
)
```

both should return `SurfaceMeshDraft` object which can be submitted:

```python
surface_mesh = surface_mesh.submit()
```

Comment (feilin-flexcompute) [reply flow360/component/surface_mesh.py]: fixed.

File: tests/data/surface_mesh/airplaneGeometry.stl
Comment (maciej-flexcompute) [inline tests/data/surface_mesh/airplaneGeometry.stl:1]: this file is not used in any test. Can you extend tests here:
https://github.com/flexcompute/Flow360/blob/develop/tests/test_surface_mesh.py

Comment (feilin-flexcompute) [reply tests/data/surface_mesh/airplaneGeometry.stl:1]: added

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**